editors/code/src/main.ts

```diff
@@ -200,3 +191,26 @@ function createCommands(): Record<string, CommandFactory> {
         revealDependency: { enabled: commands.revealDependency },
     };
 }
+
+function checkConflictingExtensions() {
+    if (vscode.extensions.getExtension("rust-lang.rust")) {
```

Comment:
We can probably drop this check in the near future (May?), it's been marked for years as deprecated and it warns on start-up.

Reply:
I prefer to keep it in this PR for now. It could be removed in next PR if we do decide to drop it.

Reply:
Yeah, I was proposing end of May or so, to mark two years of depreciation notices.